README.md

```diff
@@ -41,6 +41,14 @@ Alternatively you can provide a custom path to the properties file using `-Dprop
 
 If your Kafka is secured you will need to enable the security configuration options in your properties file
 
+## Build prerequisites
+
+To build this project you will need:
+
+- maven `mvn`
+- npm version 6.4.1 or later
+- node 10.15.0 or later
```

Comment:
The mvn install will automatically pull down Node for the build so it might be worth mentioning that you only need node installed if you want to run the UI by itself

Reply:
Agree - maybe need to move the plugin in the `pom.xml` to an `install` phase?

Reply:
Also @JQrdan there's no `npm prebuild` script

Reply:
Good point - will update

Reply:
Oddly @nictownsend your commits are appearing before mine here. Absolutley agree - the contribution is at the top level, and that calls out the commit liniting. Can we raise an issue to move the git linting there?